notebooks/how-to/04-sparse-data.ipynb

```diff
@@ -0,0 +1,654 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Assimilating sparse data\n",
+    "\n",
+    "In this demo, we'll look again at estimating the fluidity coefficient $A$ in Glen's flow law\n",
+    "\n",
+    "$$\\dot\\varepsilon = A\\tau^3$$\n",
+    "\n",
+    "from observational data for the Larsen Ice Shelf.\n",
+    "The previous tutorial showed how to solve statistical estimation problems when we can assume that the measured data $u^o$ are a continuous field defined throughout our entire domain.\n",
+    "In that setting the model-data misfit functional is\n",
```

Comment:
This tutorial is a direct follow on from `notebooks/tutorials/05-ice-shelf-inverse` so I think you should either move it to tutorials or make it clear that you're referring back to that tutorial